packages/flutter_list_bloc/lib/src/list_bloc_builder.dart

```diff
@@ -4,9 +4,9 @@ import 'package:list_bloc/list_bloc.dart';
 
 import 'types.dart';
 
-class ListBlocBuilder<T, F> extends StatelessWidget {
-  final ListCubit<T, F> Function(BuildContext context)? create;
-  final ListCubit<T, F>? cubit;
+class ListBlocBuilder<B extends ListCubit<T, F>, T, F> extends StatelessWidget {
```

Comment:
@ayushin This is the breaking change where we can put the type of the bloc when creating a ListBlocBuilder.

And in code, something like this change will look like

```Dart
DataBlocBuilder<PetDataBloc, Pet, PetReadFilter>(
          emptyBuilder: (_, __) => const Center(
            child: Text('No data'),
          ),
```